src/components/Routes/constants.tsx

```diff
@@ -1,11 +1,12 @@
 /* eslint-disable @typescript-eslint/naming-convention */
 import { lazy } from 'react';
 
-import { User } from 'contexts/UserContext/reducer';
+import { UserProfile } from 'types/User';
```

Comment:
If you're only importing types you can use the special TS import `import type`. More about it [here](https://dev.to/suryawiguna/css-position-fixed-vs-sticky-5232).

Reply:
In this case it is an interface but this article helped me a lot to understand what you want to tell me, thanks

[https://javascript.plainenglish.io/leveraging-type-only-imports-and-exports-with-typescript-3-8-5c1be8bd17fb](this)